dali/operators/image/resize/resize_attr.cc

```diff
@@ -0,0 +1,355 @@
+  .AddOptionalArg("resize_y", R"(The length of the Y dimension of the resized image.
+This option is mutually exclusive with `resize_shorter`, `resize_longer` and `size`.
+If the `resize_x` is left unspecified or 0, then the op will keep
+the aspect ratio of the original image. Negative value flips the image.)", 0.f, true)
+  .AddOptionalArg("resize_z", R"(The length of the Z dimension of the resized volume.
+This option is mutually exclusive with `resize_shorter`, `resize_longer` and `size`.
+If the `resize_x` and `resize_y` are left unspecified or 0, then the op will keep
+the aspect ratio of the original volume. Negative value flips the volume.)", 0.f, true)
+  .AddOptionalArg<vector<float>>("size", R"(The desired output size. Must be a list/tuple with the
+one entry per spatial dimension (i.e. excluding video frames and channels). Dimensions with
+0 extent are treated as absent and the output size will be calculated based on other extents
+and ``mode`` argument.)", {}, true)
+  .AddOptionalArg("mode", R"(Resize mode - one of:
+  * "default"     - image is resized to the specified size; missing extents are scaled
+                    with the average scale of the provided ones
+  * "stretch"     - image is resized to the specified size; missing extents are not scaled at all
+  * "not_larger"  - image is resized, keeping the aspect ratio, so that no extent of the
+                    output image exceeds the specified size - e.g. a 1280x720 with desired output
+                    size of 640x480 will actually produce 640x360 output.
+  * "not_smaller" - image is resized, keeping the aspect ratio, so that no extent of the
+                    output image is smaller than specified - e.g. 640x480 image with desired output
+                    size of 1920x1080 will actually produce 1920x1440 output.
+
+  This argument is mutually exclusive with `resize_longer` and `resize_shorter`)", "default")
+  .AddOptionalArg("resize_shorter", R"(The length of the shorter dimension of the resized image.
+This option is mutually exclusive with `resize_longer` and explicit size arguments
+The op will keep the aspect ratio of the original image.
+This option is equivalent to specifying the same size for all dimensions and ``mode="not_smaller"``.
+The longer dimension can be bounded by setting the `max_size` argument.
+See `max_size` argument doc for more info.)", 0.f, true)
+  .AddOptionalArg("resize_longer", R"(The length of the longer dimension of the resized image.
+This option is mutually exclusive with `resize_shorter` and explicit size arguments
+The op will keep the aspect ratio of the original image.
+This option is equivalent to specifying the same size for all dimensions and ``mode="not_larger"``.
+)", 0.f, true)
+  .AddOptionalArg<vector<float>>("max_size", R"(Limit of the output size - when resizing using
+`resize_shorter`, "not_smaller" mode or otherwise leaving some extents unspecified, some images
```

Comment:
```suggestion
`resize_shorter`, `not_smaller` mode or otherwise leaving some extents unspecified, some images
```

Reply:
These are strings (in python) - hence, double quotes, not inverted quotes (although maybe it should be `"not_smaller"`?)

Reply:
Run CI and check generated docs, if that is what you really want.